src/test/java/com/hubspot/jinjava/lib/filter/PrettyPrintFilterTest.java

```diff
@@ -94,6 +96,10 @@ public int getBar() {
       return 123;
     }
 
+    public ZonedDateTime getDate() {
+      return ZonedDateTime.of(2023, 12, 11, 0, 0, 0, 0, ZonedDateTime.now().getZone());
+    }
```

Comment:
Is this method used?

Reply:
Yes, this tests that `ZonedDateTime` (the problem type) is being serialized successfully.